Makefile

```diff
@@ -1,27 +1,32 @@
 SENTRY_PATH := `python -c 'import sentry; print sentry.__file__.rsplit("/", 3)[0]'`
 UNAME_S := $(shell uname -s)
 
-develop: setup-git install-yarn
-	pip install "pip>=9,<10"
-	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional]
-	pip install -e ".[tests]"
+PIP_VERSION := 19.1.1
```

Comment:
Latest is `19.2.1` so why not that one?

Reply:
It's simply because we have 19.1.1 in a lot of other places.

I don't know if it's common to do this, but I think we should serve some highly available strings on the web like `http://chromedriver.storage.googleapis.com/LATEST_RELEASE_77`. Something like `https://cfg.sentry.io/PYTHON_PIP_VERSION`. That way, it's not nearly as painful to bump/change stuff like this across all our repositories, and we have an easy-to-find single source of truth. cc @mattrobenolt

Reply:
That's an interesting idea. We may look into this. A better thing would be to consolidate all these separate builds and make files under one roof tho as there are more things to diverge.